Don't fetch promu form internet. This is just a build-dependency

Index: Makefile.common
--- Makefile.common.orig
+++ Makefile.common
@@ -261,11 +261,7 @@ common-docker-manifest:
 promu: $(PROMU)
 
 $(PROMU):
-	$(eval PROMU_TMP := $(shell mktemp -d))
-	curl -s -L $(PROMU_URL) | tar -xvzf - -C $(PROMU_TMP)
-	mkdir -p $(FIRST_GOPATH)/bin
-	cp $(PROMU_TMP)/promu-$(PROMU_VERSION).$(GO_BUILD_PLATFORM)/promu $(FIRST_GOPATH)/bin/promu
-	rm -r $(PROMU_TMP)
+	@true
 
 .PHONY: proto
 proto:
